hyper/pkg/proto/hyper.proto

```diff
@@ -36,27 +36,6 @@ message DesktopStopRequest {
 
 message DesktopStopResponse {}
 
-message DesktopListRequest {}
-
-message DesktopListResponse {
-    repeated string ids = 1;
-}
-
-message DesktopXMLGetRequest {
-    string id = 1;
-}
-
-message DesktopXMLGetResponse {
-    string xml = 1;
-}
-
-message DesktopMigrateRequest {
-    string id = 1;
-    string hyper = 2;
-}
-
-message DesktopMigrateResponse {}
-
 message DesktopResumeRequest {
```

Comment:
`DesktopResume` and `DesktopSuspend` should be in the reverse order

Reply:
Why does this affect?

Reply:
It follows a "execution" logic: To be able to resume a desktop, it first needs to be paused

Reply:
ok